README.md

```diff
@@ -1 +1,46 @@
-# dat
+# dat: Delta Acceptance Testing
+
+Warning: this project is incubating 🚧 🛠.  Any changes are possible in the future, including full deletion of this repo.
+
+This repo contains Delta Lake reference tables.  It specifies functionality that the [Delta connectors](https://github.com/delta-io/connectors) should be able to implement.  The Delta Lake connectors will use these reference tables in integration tests to demonstrate they can perform the requisite read / write operations on Delta Lakes.
```

Comment:
nit: this isn't only for that specific Delta connectors repo. This is for any connector for delta. including even Delta Lake on Spark.